intf/cnsr_intf.lua

```diff
@@ -145,8 +146,21 @@ it runs all the time in the background of VLC, finds the relevant tag and execut
 function looper()
 	local next_loop_time = vlc.misc.mdate()
 	local loop_counter = 0
+	local current_uri = nil
+	
 	while true do
 		if vlc.volume.get() == -256 then break end  -- inspired by syncplay.lua; kills vlc.exe process in Task Manager
+		
+		current_item = vlc.playlist.get(vlc.playlist.current()) -- if a new video is playing, then we want to get the new config
+		if current_item ~= nil and current_uri ~= current_item.path then
+			current_uri = current_item.path
+			tag_by_end_time_index = 1
+			tag_index = 1
+			loop_counter = 0
+			log("new video")
+			sleep(SLEEP_WHILE_PARSING) -- wait for cnsr_ext to finish parsing the new cnsr file
```

Comment:
Great idea!
but maybe use vlc.misc.mwait(vlc.misc.mdate() + <microseconds>) which doesn't hog the CPU :-P